lib/hooks.dart

```diff
@@ -151,11 +151,7 @@ void useEffect(dynamic Function() sideEffect, [List<Object> dependencies]) {
     return jsUndefined;
   });
 
-  if (dependencies != null) {
-    return React.useEffect(wrappedSideEffect, dependencies);
-  } else {
-    return React.useEffect(wrappedSideEffect);
-  }
```

Comment:
I assume these changes mean passing in `null` for dependencies is the same as not passing it?